app/views/state_file/questions/income_review/edit.html.erb

```diff
@@ -27,5 +27,14 @@
     </div>
   <% end %>
 
+  <% if current_intake.direct_file_data.fed_ssb > 0 || current_intake.direct_file_data.fed_taxable_ssb > 0 %>
+    <div class="white-group">
+      <p class="text--bold spacing-below-0"><%= t(".ssa_title") %></p>
```

Comment:
i'm following the markup structure from the card above, but semantically i think this should be an `h2` or `h3` rather than a `p`.

Reply:
low key agree but no strong opinion